src/Panel.php

```diff
@@ -60,7 +60,10 @@ public function getPanel()
 		$installed = $this->decode($lockFile);
 
 		if ($this->error === NULL) {
+			$required = array_filter($required);
+			$installed = array_filter($installed);
```

Comment:
Sure you have an `"packages-dev": null` in your `composer.lock`? I can't reproduce it.